From ad6a8d275ab588417af8d215152c57a4bf70d318 Mon Sep 17 00:00:00 2001
From: Paul Gortmaker <paul.gortmaker@windriver.com>
Date: Tue, 9 Mar 2010 18:01:51 -0500
Subject: [PATCH] multicore: delete the MIPC specific changes.

These are moving to the MIPC template instead of being deployed
globally.  From there, they will be assessed as to whether
they are needed and whether they can be merged into part of
the MIPC module functionality.

Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>
---
 arch/powerpc/kernel/vbi/wrhv.c |    5 -----
 drivers/serial/serial_core.c   |    9 ---------
 2 files changed, 0 insertions(+), 14 deletions(-)

diff --git a/arch/powerpc/kernel/vbi/wrhv.c b/arch/powerpc/kernel/vbi/wrhv.c
index 3fab15c..843a09a 100644
--- a/arch/powerpc/kernel/vbi/wrhv.c
+++ b/arch/powerpc/kernel/vbi/wrhv.c
@@ -325,11 +325,6 @@ void wrhv_mapping(void)
 			 sharedMemoryRegionsConfigAddress,
 			 (unsigned long)wr_config->
 			 sharedMemoryRegionsConfigAddress, _PAGE_RAM);
-#ifndef CONFIG_WRHV_MILS
-	/* MIPC */
-	map_page(0xfd000000, 0xfd000000, _PAGE_IO);
-#endif
-	/* devices mapped by the hypervisor xml coqnfiguration */
 
 	return;
 }
diff --git a/drivers/serial/serial_core.c b/drivers/serial/serial_core.c
index 7ec0192..bcdc4b1 100644
--- a/drivers/serial/serial_core.c
+++ b/drivers/serial/serial_core.c
@@ -2499,15 +2499,6 @@ int uart_add_one_port(struct uart_driver *drv, struct uart_port *port)
 		lockdep_set_class(&port->lock, &port_lock_key);
 	}
 
-#if defined(CONFIG_WRHV) && !defined(CONFIG_WRHV_MILS)
-#define __MIPC_MSD_MAJOR	236
-	if (!uart_console(port) && (drv->major != __MIPC_MSD_MAJOR)) {
-		ret = -EBUSY;
-		goto out;
-	}
-#undef __MIPC_MSD_MAJOR
-#endif
-
 	uart_configure_port(drv, state, port);
 
 	/*
-- 
1.7.0

